src/infinite-list/index.ts

```diff
@@ -4,4 +4,20 @@ import List from '../list';
 @tag('gb-infinite-list', require('./index.html'), require('./index.css'))
 class InfiniteList extends List {}
 
+namespace InfiniteList {
+  export interface Props extends Tag.Props {
```

Comment:
`extends List.Props` since `InfiniteList` extends `List`. That also means that you don't need to redefine `items` or `layout` or anything else that's defined in `List.Props`.